package Exception;

public class Demo 
{

	public static void main(String[] args) 
	{
		int i = 5;
		int j = 0;
		int k;
		try
		{
		k = i/j;
		}
		catch(Exception e)
		{
			System.out.println("Error");
		}
		//System.out.println(k);
	}

}
